sklearn/feature_extraction/dict_vectorizer.py

```diff
@@ -85,6 +89,17 @@ class DictVectorizer(BaseEstimator, TransformerMixin):
     True
     >>> v.transform({'foo': 4, 'unseen_feature': 3})
     array([[0., 0., 4.]])
+    >>> D2 = [{'foo': '1', 'bar': '2'}, {'foo': '3', 'baz': '1'}, {'foo': ['1', '3']}]
```

Comment:
The purpose of each additional example needs to be clear. Put a line of explanation before each